find_free_space_action_server:
  ros__parameters:
    topic_name_global_costmap: '/global_costmap/costmap_raw'
    service_name_check_car_passble: '/check_car_passable'
    topic_name_footprint: '/global_costmap/published_footprint'
    redundancy_distance: 0.5
    search_interval: 0.5
    search_radius: 4.0





